greenheart/tools/optimization/openmdao.py

```diff
@@ -305,90 +302,3 @@ def compute(self, inputs, outputs):
 
     def setup_partials(self):
         self.declare_partials('*', '*', method='fd', form='forward')
-
```

Comment:
Why are the electrolyzer changes I already made showing up in this PR?

Reply:
That's a great question. It looks like I accidentally merged develop in at some point. I'll update that.